server/routes/activities/create-an-activity/journey.ts

```diff
@@ -44,6 +44,7 @@ export type CreateAnActivityJourney = {
   }>
   startDate?: string
   endDateOption?: string
+  removeEndDate?: string
```

Comment:
Not sure you need `removeEndDate` ?

Reply:
yea you're right... don't think it's necessary, removed